app/models/article.rb

```diff
@@ -30,6 +30,10 @@ class Article < ApplicationRecord
   counter_culture :user
   counter_culture :organization
 
+  MAX_USER_MENTIONS = 7 # Explicitly set to 7 to accommodate DEV Top 7 Posts
```

Comment:
The [design/implementation](https://github.com/forem/rfcs/blob/main/text/0022-expand-at-mention-to-posts.md#designimplementation) section of the related RFC mentions this: 
> There needs to be a cap/limit on how many people can be @-mentioned in a post in order to prevent spam. I would propose a limit of 10.

I initially set off to make the limit of @-mentioned users in a post as 10...but the more that I thought about it, and the more I looked at [what we do with comments](https://github.com/forem/forem/pull/12923), it seemed kind of weird to me to treat posts and comments differently for no real reason. So, you'll notice that this PR sets the max limit of mentions in a post to `7`, just like comments; we can always change this later or now if folks want, but I just went with my gut on this one!

Reply:
This makes a _lot_ of sense. As a user, I was surprised that comments and posts were treated differently. Should we extract a single constant (or `SiteConfig` setting) for this?

Reply:
@jgaskins Yeah, that's definitely a possibility. The only reason I didn't extract is right _now_ was because I wasn't entirely sure if folks would want to bump up the number of @-mentions in posts for some reason (maybe some reason I didn't think of!); plus, the RFC's "definition of done" included 10 @-mentions as the cap, so I didn't want to extract anything without first getting a sense of whether we would ever want to keep the number of mentions separate between the two text types (comments and articles). 

I also wonder if we'll know the answer to these questions better once we've got the "@-mention notification" functionality working for posts; maybe that would be the right time to assess if extracting this into either a constant or into the `SiteConfig` makes sense or not? 🤔 The other thing about the `SiteConfig` is making the decision about whether we want Forem admins to be able to configure the "max number of user mentions" in articles versus comments (but that's more of a product decision, ultimately).

I kinda lean towards keeping them separate for now, finishing the feature for RFC 22, and then coming back to see whether there is a use case for keeping the cap separate; if there isn't, then we can just extract it -- either into a constant or into a configurable thing on `SiteConfig`, if that makes the most product sense.

How do you feel about that?

Reply:
It sounds great. Apologies, a lot of this hinged on the fact that I forgot that article mentions aren't on the level with comment mentions yet. :100:

Reply:
@jgaskins No worries! Glad you're asking questions and making me think about what I'm doing :)

I left some `TODO:` comments [in the latest commit](https://github.com/forem/forem/pull/13259/commits/3e23a8cd47cf25d094a175925639a58f269dc553) to circle back to this after the RFC is completed!